test/helpers/data-source-factory.js

```diff
@@ -10,8 +10,14 @@ const connector = require('../..');
 const DataSource = require('loopback-datasource-juggler').DataSource;
 const extend = require('util')._extend;
 
+// Allow urls with hardcoded mapname or dynamic mapname based on model names
+// (ie. no trailing `/default` in the url).
+let url = process.env.EXTREME_SCALE_URL;
+if (/\/wxsdata\/v1\/grids\/[^\/]+\/default$/.test(url))
```

Comment:
`process.env.EXTREME_SCALE_URL` may be not set (see the assert on [L24/L30](https://github.com/strongloop/loopback-connector-kv-extreme-scale/pull/12/files#diff-8e5b426819f02c14155e847c4e36d632L24) below).

```js
if (url && /\/wxsdata\/v1\/grids\/[^\/]+\/default$/.test(url))
```